pyiron_continuum/schroedinger/schroedinger.py

```diff
@@ -26,9 +26,12 @@
 
 
 # TODO: Convert to pyiron units
-HBAR = 1  # EV_TO_U_ANGSTROMSQ_PER_SSQ * physical_constants['Planck constant in eV/Hz'][0] / (2 * np.pi)
-M_EL = 1  # physical_constants['electron mass in u'][0]
+HBAR = physical_constants['reduced Planck constant in eV s'][0]
 KB = physical_constants['Boltzmann constant in eV/K'][0]
+# conversion factor to convert the units of all terms in the Schroedinder equation are in eV. More documentation
+# in _hamiltonian of the TISE class.
+EV_S_PER_ANG_SQUARED_PER_AMU_IN_EV = 9.64853322e27
```

Comment:
Name is wrong, it's (eV s)^2. Writing 'squared' in all places will be too long though, even for me.

What about `EV2_S2_PER_ANG2_AMU2_IN_EV`?

Reply:
It's supposed to be (eV s / A)^2 / AMU. 

> What about EV2_S2_PER_ANG2_AMU2_IN_EV

Reads okay.

Reply:
ah, yeah, `AMU` not `AMU2`